angular-workspace/.storybook/main.ts

```diff
@@ -63,7 +63,7 @@ const config: StorybookConfig = {
     return config;
   },
   framework: {
-    name: getAbsolutePath("@storybook/angular"),
+    name: "@storybook/angular",
```

Comment:
Ik zag hier al een tijdje TypeScript compile errors in mijn console voorbijkomen. En het werkt zonder `getAbsolutePath()` net zo goed 🤷 

In alle weggehaald en getest.